source/management_and_operations/multivm_service_management/appflow_use_cli.rst

```diff
@@ -336,6 +340,47 @@ When a Service fails during a deployment, undeployment or scaling operation, the
 
 In order to delete all the services in ``DONE`` state, to free some space in your database, you can use the command ``oneflow purge-done``.
 
+Instantiation of roles with VMs on hold
+--------------------------------------------------------------------------------
+
+VMs of a Service can be instances on hold with the ``on_hold`` parameter set to true in the Service Template.
+
+.. code-block:: javascript
+
+    {
+      "name": "my_service",
+      "deployment": "straight",
+      "on_hold": true|false,
+      "roles": [
+        {
+          ...
+        }
+      ]
+    }
+
+This option can also be set at the Role level, where only one specific Role is instantiated in ``HOLD``, instead of the whole service. Please see the following example:
+
+.. code-block:: javascript
+
+    {
+      "name": "my_service",
+      "deployment": "straight",
+      "roles": [
+        {
+          "name": "frontend",
+          "vm_template": 0,
+          "on_hold": true|false
+        },
+        ...
+      ]
+    }
+
+Once you want to release the Roles, you can use the ``oneflow release`` command to release the Service:
+
+.. prompt:: bash $ auto
+
+    $ oneflow release <SERVICE_ID>
+
```

Comment:
Please review [API](https://docs.opennebula.io/6.2/integration_and_development/system_interfaces/appflow_api.html) docs to see if some information needs to be added

Reply:
I've already added the information related to hold / release functionality to API docs